source-code/cli/src/commands/machine/translate.ts

```diff
@@ -0,0 +1,151 @@
+import { query } from "@inlang/core/query"
+import { setupConfig } from "@inlang/core/config"
+import { initialize$import, InlangEnvironment } from "@inlang/core/environment"
+import fs from "node:fs"
+import { Command } from "commander"
+import { countMessagesPerLanguage, getFlag, log } from "../../utilities.js"
+import type { Message } from "@inlang/core/ast"
+import { rpc } from "@inlang/rpc"
+
+export const translate = new Command()
+	.command("translate")
+	.description("Machine translate all resources.")
+	.action(translateCommandAction)
+
+async function translateCommandAction() {
+	// Set up the environment functions
+	const env: InlangEnvironment = {
+		$import: initialize$import({
+			// @ts-ignore TODO: use @inlang-git/fs
+			fs: fs.promises,
+			fetch,
+		}),
+		// @ts-ignore TODO: use @inlang-git/fs
+		$fs: fs.promises,
+	}
+
+	const filePath = process.cwd() + "/inlang.config.js"
+
+	if (!fs.existsSync(filePath)) {
+		log.error("No inlang.config.js file found in the repository.")
+		return
+	} else {
+		log.info("✅ Using inlang config file at `" + filePath + "`")
+	}
+
+	// Get the content of the inlang.config.js file
+	const file = fs.readFileSync(filePath, "utf-8")
+
+	const config = await setupConfig({
+		module: await import("data:application/javascript;base64," + btoa(file.toString())),
+		env,
```

Comment:
You can directly import the config instead of a dynamic import (which is a workaround for the editor). 

```ts

const module = await import(filePath)
```

Reply:
works!